providers/edge/src/airflow/providers/edge/worker_api/routes/worker.py

```diff
@@ -149,6 +149,9 @@ def register(
     worker: EdgeWorkerModel = session.scalar(query)
     if not worker:
         worker = EdgeWorkerModel(worker_name=worker_name, state=body.state, queues=body.queues)
+    worker.maintenance_comment = (
+        "" if worker.state != EdgeWorkerState.OFFLINE_MAINTENANCE else worker.maintenance_comment
```

Comment:
To be consistent with other cases I assume this should be set to `None` as well?
```suggestion
        None if worker.state != EdgeWorkerState.OFFLINE_MAINTENANCE else worker.maintenance_comment
```
Otherwise I am thinking a bit if this is needed at all. Either the worker was in maintenance, then you can keep the comments, or it is not - then it was None before. And a fresh / new worker would get None as default?

Reply:
Yes, this is not needed at all